modules/mwaa/iam.tf

```
@@ -0,0 +1,31 @@
+module "iam_policy" {
+  source  = "cloudposse/iam-policy/aws"
+  version = "0.3.0"
```

newer version is out
```suggestion
  version = "0.4.0"
```